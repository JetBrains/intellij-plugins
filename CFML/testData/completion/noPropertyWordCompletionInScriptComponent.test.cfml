component {
  aaa = 12;
  prop<caret>
}