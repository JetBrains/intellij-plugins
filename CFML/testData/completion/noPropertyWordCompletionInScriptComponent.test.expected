component {
  aaa = 12;
  prop
}